packages/react/src/side-nav/SideNav.stories.tsx

```diff
@@ -47,3 +54,11 @@ export const OnBodyAlt: Story = {
 		</Box>
 	),
 };
+
+export const always: Story = {
```

Comment:
Clearer name (and correct capitalistion)?
```suggestion
export const SubLevelsAlwaysOpen: Story = {
```